spring/src/main/kotlin/de/codecentric/hikaku/converters/spring/SpringConverter.kt

```diff
@@ -51,17 +52,21 @@ class SpringConverter(private val applicationContext: ApplicationContext) : Abst
                     headerParameters = mappingEntry.value.hikakuHeaderParameters(),
                     matrixParameters = mappingEntry.value.hikakuMatrixParameters(),
                     produces = mappingEntry.produces(),
-                    consumes = mappingEntry.consumes()
+                    consumes = mappingEntry.consumes(),
+                    deprecated = mappingEntry.value.method.isAnnotationPresent(Deprecated::class.java)
+                            || mappingEntry.value.method.declaringClass.isAnnotationPresent(Deprecated::class.java)
```

Comment:
I think it would be nice to extract that into a `isEndpointDeprecated` function.